src/components/presentational/Grid/Grid.tsx

```diff
@@ -31,12 +31,9 @@ export function Grid(props: GridProps) {
     }
 
     let style = { ...props.style };
-    if (props.gap) {
-        style.gap = props.gap + "px";
-    } else {
-        style.gap = "16px";
-    }
-    return <GridContext.Provider value={{ gap: props.gap || 16 }}>
+    let gap = (props.gap || props.gap === 0) ? props.gap : 16;
```

Comment:
This just fixed a bug where if gap was specified to be 0, it would default to 16